capture-sdk/sdk/src/main/res/values/styles.xml

```diff
@@ -1,6 +1,6 @@
 <resources xmlns:tools="http://schemas.android.com/tools">
 
-    <style name="Root.GiniCaptureTheme" parent="Theme.Material3.DayNight">
+    <style name="Root.GiniCaptureTheme" parent="Theme.Material3.DayNight.NoActionBar">
```

Comment:
[Request] Please keep the action bar. We need it to check that `Scan an invoice or a QR code` is shown in the action bar on tablets.

Reply:
Done